### PR TITLE
Clean up handling of addresses in testvm

### PR DESCRIPTION
**Pull request by stefwalter**
Besides cleanup, and getting ready for IPv6, this should fix some of the macaddr races involved.

### CODE REVIEW

File: test/guest/network-cockpit.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<network>
+<network ipv6='yes'>
```
Comment (dperpeet) [inline test/guest/network-cockpit.xml]: do we have mechanics in place on the test runners to recreate the network if the definition changed?

Comment (stefwalter) [inline test/guest/network-cockpit.xml]: No and that's why I want to get this in place now. And then at some point we can actually make use of this in the future.

**Comment by dperpeet** [comment]
looks good!

**Comment by stefwalter** [comment]
Filed #3102 for this failure: http://files.cockpit-project.org/logs/pull-3083-6b16cff5-fedora-23-x86_64/log

**Comment by stefwalter** [comment]
Filed #3099 for this failure: http://fedorapeople.org/groups/cockpit/logs/pull-3083-6b16cff5-rhel-7-x86_64/

**Comment by stefwalter** [comment]
This is blocking my work on fedora-atomic. Could we merge this?

**Pull request closed without merging**